node.go

```diff
@@ -73,7 +79,7 @@ func (n *Node) Close(optArgs ...CloseOpts) error {
 		}
 	}
 
-	n.refreshTicker.Stop()
```

Comment:
The ticker is not part of the node struct anymore but local to the goroutine. If the goroutine exits, the ticker goes away as well.